### ISSUE TITLE
[runtimes] [spine-ts] [spine-threejs] ImageBitmap not supported in ThreeJsTexture

### ISSUE THREAD

**Issue by mrzlxa** [EsotericSoftware/spine-runtimes#2108]
Was trying to create my own ThreeJsTexture. Faced an error, because of a check in constructor.

```
export class ThreeJsTexture extends Texture {
    constructor(image) {
        super(image);


// SUPPOSEDLY WRONG CHECK?
        if (image instanceof ImageBitmap)
            throw new Error("ImageBitmap not supported.");
// !SUPPOSEDLY WRONG CHECK?


        this.texture = new THREE.Texture(image);
        this.texture.flipY = false;
        this.texture.needsUpdate = true;
        }
}
```

Thought this change is weird, found a commit with it here 
https://github.com/EsotericSoftware/spine-runtimes/commit/e4372e5329f9fe66b7b97e939d2fa800b04ce99a

This introduced "TS strict mode" which should not probably lead to this change.  

Does this check have to be there? 

Also, given ImageBitmap is supported in AssetManagerBase.js ([link to loadTexture() method](https://github.com/EsotericSoftware/spine-runtimes/blob/4.1/spine-ts/spine-core/src/AssetManagerBase.ts#L136)) this looks suspiciously wrong. 

Thanks!

**Comment by dan-playboomsports** [comment]
We're also currently seeing this issue as well.  It's breaking our apps on iOS 14.  Is there any update or status on a fix for this bug?

**Comment by badlogic** [comment]
This has been resolved and the fix is in both the 4.1 and 4.2-beta branch. You can update to spine-threejs 4.1.20 via NPM (or 4.2.5 for the 4.2-beta branch).

**Comment by mrzlxa** [comment]
Awesome! thanks!
